libraries/AP_BoardConfig/AP_BoardConfig_CAN.cpp

```diff
@@ -83,6 +84,28 @@ const AP_Param::GroupInfo AP_BoardConfig_CAN::var_info[] = {
     AP_SUBGROUPINFO(_drivers[2], "D3_", 6, AP_BoardConfig_CAN, AP_BoardConfig_CAN::Driver),
 #endif
 
+    // @Param: SLCAN_RT
+    // @DisplayName: SLCAN Route
+    // @Description: CAN Driver ID to be routed to SLCAN, 0 means no routing
+    // @Range: 0 3
+    // @User: Advanced
+    AP_GROUPINFO("SLCAN_RT", 7, AP_BoardConfig_CAN, _slcan_rt, 1),
+
+    // @Param: SLCAN_SR
+    // @DisplayName: SLCAN Serial Port
+    // @Description: Serial Port ID to be used for temporary SLCAN iface, -1 means no temporary serial
+    // @Range: 0 5
+    // @User: Advanced
+    AP_GROUPINFO("SLCAN_SR", 8, AP_BoardConfig_CAN, _slcan_sr, -1),
+
+
+    // @Param: SLCAN_TO
+    // @DisplayName: SLCAN Timeout
+    // @Description: Duration of inactivity after which SLCAN in seconds
+    // @Range: 0 255
+    // @User: Advanced
+    AP_GROUPINFO("SLCAN_TO", 9, AP_BoardConfig_CAN, _slcan_to, 0),
```

Comment:
it is a uint8, which means max 127 with param protocol. Maybe make it a uint16?
Also, the variable name should be _slcan_timeout, or even better put all slcan vars in a slcan structure, and call it timeout

same with other variable names "_slcan_sr" isn't an obvious name. We need to keep param names short, but not variable names.
For the parameter names, I think CAN_SLCAN_TIMOUT would be better (it fits in 16), and CAN_SLCAN_SERNUM would be better. For the route, I think CAN_SLCAN_CPORT would be better (can port)

Reply:
@tridge fixed in commit https://github.com/ArduPilot/ardupilot/pull/10171/commits/79c9126477114b1309430d6e997af5f850ec8ec6